lib/mappers/esdn_mapper.py

```diff
@@ -0,0 +1,331 @@
+from akara import logger
+from dplaingestion.utilities import iterify
+from dplaingestion.selector import exists, getprop
+from dplaingestion.mappers.oai_mods_mapper import OAIMODSMapper
+
+class EsdnMapper(OAIMODSMapper):
+    def __init__(self, provider_data):
+        super(EsdnMapper, self).__init__(provider_data)
+
+    def map_collection(self):
+        ret_dict = {"collection": getprop(self.provider_data, "collection")}
+
+        prop = self.root_key + "relatedItem"
+
+        if exists(self.provider_data, prop):
+            related_items = iterify(getprop(self.provider_data, prop))
+            host_types = [item for item in related_items if
```

Comment:
`host_types` could benefit from being renamed to something like `collection_items`, because it looks like it's really enumerating something it's calling items.